File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntSpreadsheetRefTokenMap.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import org.jetbrains.annotations.Nullable;

import java.util.Map;

import com.neome.api.meta.base.Types.MetaIdSpreadsheetRef;
import com.neome.api.meta.base.dto.StudioBase;

@SuppressWarnings("unused")
public class StudioEntSpreadsheetRefTokenMap extends StudioBase
{
  @Nullable
  public Map<MetaIdSpreadsheetRef, String> refTokenMap;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioEntSpreadsheetRefTokenMap.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import java.util.Map
import com.neome.api.meta.base.Types.MetaIdSpreadsheetRef
import com.neome.api.meta.base.dto.StudioBase

interface StudioEntSpreadsheetRefTokenMap : StudioBase
{
  val refTokenMap: Map<MetaIdSpreadsheetRef, String>?
}